src/FSharpx.Collections.Experimental/paket.template

```diff
@@ -28,4 +28,9 @@ dependencies
   framework: netstandard20
     FSharp.Core 4.3.4
 files
-    ../../bin/FSharpx.Collections.Experimental ==> lib
\ No newline at end of file
+    ../../bin/FSharpx.Collections.Experimental/net45/FSharpx.Collections.Experimental.dll ==> lib/net45
```

Comment:
Previous solution included FSharp.Core in the net45 package. Approach from https://github.com/mbraceproject/FsPickler/blob/master/src/FsPickler/paket.template#L42-L47